kpops/utils/yaml.py

```diff
@@ -15,6 +15,17 @@
 log = logging.getLogger("Yaml")
 
 
+def touch_yaml_file(file_name, dir_path) -> Path:
```

Comment:
is it worth it to have this as a function? this could be a one-liner pretty much
but if so, please add missing type annotations

Reply:
Removed